pom.xml

```diff
@@ -18,6 +18,13 @@
         </maven.checkstyle.plugin.configLocation>
     </properties>
 
```

Comment:
add an empty line below or remove this empty line to separate dependencies

Reply:
I have added empty lines to separate the dependecies